### PR TITLE
Essence printer

### PR DESCRIPTION
**Pull request by MadmanMartian**
Adds the Essence Printer, sprite and idea courtesy of Megautham on discord

You click it, it binds your essence to it, not unlike a phylactery.

On death, it creates a duplicate of your body over 30 seconds to a minute. Problem is, ANY ghost can take it over, and it halves the health of the clone.

### CODE REVIEW

**Comment by Arthurdentist** [comment]
when you say take over, do you mean it will poll everybody in dedchat like a diona nymph?

**Comment by Pathid** [comment]
Nope, no polling. It just waits for the first ghost to click on it.

File: code/modules/research/xenoarchaeology/artifact/artifact_essenceprinter.dm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	for(var/datum/language/L in R.languages)
+		H.add_language(L.name)
+	H.real_name = H.dna.real_name
+	H.flavor_text = H.dna.flavor_text
+
+	H.maxHealth = round(previous.maxHealth/2)
+	spawn(rand(30 SECONDS,60 SECONDS))
+		do_flick(src,"Essence_imprinter_scan_complete",8)
+		icon_state = "Essence_imprinter_idle"
+		ready = TRUE
```
Comment (Pathid) [inline code/modules/research/xenoarchaeology/artifact/artifact_essenceprinter.dm:83]: Noticed you don't set this back to FALSE anywhere.

Comment (MadmanMartian) [reply code/modules/research/xenoarchaeology/artifact/artifact_essenceprinter.dm:83]: good catch

**Review by Intigracy — DISMISSED** [review]
confrictu

**Pull request merged**